### ISSUE TITLE
Remove dist.zip from repository

### ISSUE THREAD

**Issue by shikazu** [Semantic-Org/Semantic-UI#4043]
I've got some major issues with the search api, since i used the files from the dist.zip in your repository.

They are old files with version of 2.0.0, while the master repository is currently at 2.1.7. 

Just mentioning this, because others could get the same issue. 

Thank you!

**Comment by jlukic** [comment]
Where is this `dist.zip`? I'm not seeing it in the repo

**Comment by shikazu** [comment]
Okay sorry, i mislead you. The download I used was from the documentation "Getting Started".

There is a link to a github archive called "master", therefore i thought it was ne master repo, but actually is the archive master.

Link to doumentation site: http://semantic-ui.com/introduction/getting-started.html

I used the link found here: http://imgur.com/5Geqkam

Checked again right now, it's in there.
